deliver/lib/deliver/upload_metadata.rb

```diff
@@ -111,6 +111,7 @@ def upload(options)
       end
 
       v.release_on_approval = options[:automatic_release]
+      v.toggle_phased_release(enabled: !!options[:phased_release])
```

Comment:
What if:

- iTunes Connect: you have phased releases enabled
- _deliver_: You don't set anything manually and just use the default

I'd think that this would then disable phased releases on iTunes Connect, right?